server/boot/user.js

```diff
@@ -235,6 +238,39 @@ module.exports = function(app) {
     });
   }
 
+  function getUpdateEmail(req, res) {
+    if (!req.user) {
+      return res.redirect('/');
+    }
+    return res.render('account/email-update', {
+      title: 'Update your Email'
+    });
+  }
+
+  function postUpdateEmail(req, res, next) {
```

Comment:
These api endpoints should be added using loopbacks build in `remoteMethods`
see https://docs.strongloop.com/display/APIC/Remote+methods
and https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/common/models/user.js#L265